src/matrix/inversewishart.jl

```diff
@@ -15,12 +15,15 @@ function InverseWishart{T<:Real}(df::T, Ψ::AbstractPDMat{T})
     p = dim(Ψ)
     df > p - 1 || error("df should be greater than dim - 1.")
     c0 = _invwishart_c0(df, Ψ)
-    R = promote_type(T, typeof(c0))
-    _, prom_Ψ = promote_eltype(one(R), Ψ)
+    R = Base.promote_eltype(T, c0)
+    prom_Ψ = convert(AbstractArray{R}, Ψ)
```

Comment:
Why is this converting to an abstract type rather than a concrete type like simply `Array{R}`?